packages/nimbus/src/components/nimbus-provider/nimbus-provider.tsx

```diff
@@ -40,11 +41,19 @@ export function useColorScheme() {
   return colorScheme;
 }
 
-export function NimbusProvider({ children, ...props }: ColorModeProviderProps) {
+export function NimbusProvider({
+  children,
+  locale,
+  ...props
+}: ColorModeProviderProps & { locale?: string }) {
+  // The provider can accept an optional locale prop, ex from App-Kit
+  // then we fallback first to the navigator.language, then to "en-US"
+  const resolvedLocale = locale ?? navigator.language ?? "en-US";
```

Comment:
iirc, the `I18nProvider` handles fallback for the locale [internally](https://github.com/adobe/react-spectrum/blob/114f89071012a734987588a9466de8a7c4d38a16/packages/%40react-aria/i18n/src/context.tsx#L31), so all we need is the locale passed to the provider (from appkit or wherever else)